alpaca/trading/client.py

```diff
@@ -30,8 +44,131 @@ def __init__(
         )
 
     def submit_order(self, order_data: OrderCreationRequest) -> Order:
-        """ """
+        """Creates an order to buy or sell an asset.
+
+        Args:
+            order_data (OrderCreationRequest): The request data for creating a new order.
+
+        Returns:
+            Order: The resulting submitted order.
+        """
         data = order_data.to_request_fields()
         response = self.post("/orders", data)
 
         return Order(**response)
+
+    def get_orders(self, filter: Optional[GetOrdersRequest] = None) -> List[Order]:
+        """
+        Returns all orders. Orders can be filtered by parameters.
+
+        Args:
+            filter (Optional[GetOrdersRequest]): The parameters to filter the orders with.
+
+        Returns:
+            List[Order]: The queried orders.
+        """
+        # checking to see if we specified at least one param
+        params = filter.dict() if filter is not None else {}
+
+        if "symbols" in params and type(params["symbols"]) is List:
+            params["symbols"] = ",".join(params["symbols"])
+
```

Comment:
this transformation should probably be moved into `GetOrdersRequest` itself with an explanation about how the API doesn't support proper list style url params but instead a comma separated list

Reply:
I think it will be confusing ƒor users because we will need to change the symbols field to

`symbols: Optional[str]`

It may not be obvious that they can pass it in as a list of symbols.